### PR TITLE
update webpack-dev-server and shakapacker to allow transitive dependency on path-to-regexp 0.1.10, resolving dependabot high severity alert

### PR DESCRIPTION
**Pull request by mrotondo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,9 @@
     "jquery-mask-plugin": "^1.14.16",
     "jquery-ui": "^1.13.2",
     "mini-css-extract-plugin": "^2.7.6",
+    "prop-types": "^15.7.2",
     "regenerator-runtime": "^0.13.7",
-    "shakapacker": "7.1.0",
+    "shakapacker": "7.2.3",
```
Comment (mpidcock) [inline package.json:28]: any reason to not have "^7.2.3"?

**Review by mpidcock — APPROVED** [review]
Fine how it is, but if we can avoid locking to a specific version that would be nice

File: package.json (hunk 1 above)
Comment (mrotondo) [reply package.json:28]: The version derived from the yarnfile/package.json and the gemfile need to match, and I don't trust the process of their respective ~/^ operators to keep the two versions in sync. I don't know why the version was previously hardcoded, but I assume it was a similar reason.

**Comment by mrotondo** [comment]
Note to self: don't merge this, it seems very broken on heroku. Issues with flipper_features not being a table in the DB, and with css/js loading?

**Pull request merged**